### ISSUE TITLE
Question about training process

### ISSUE THREAD

**Issue by jelleopard** [xmeng525/RealTimeDenoisingNeuralBilateralGrid#16]
Thanks for your great work.
In the training process, I have a problem need you help.

> Except for the first epoch, the psnr and loss are the same in all other epochs. What is the problem?

`
Start Training: 

Epoch 0     Train,      psnr 25.85412068     loss = 0.04045784
Epoch 0     Valid,      psnr 23.70074201     loss = 0.05095885
best model saved
Epoch 1     Train,      psnr 24.44699055     loss = 0.04801134
Epoch 1     Valid,      psnr 23.70074201     loss = 0.05095885
Epoch 2     Train,      psnr 24.44699055     loss = 0.04801134
Epoch 2     Valid,      psnr 23.70074201     loss = 0.05095885
Epoch 3     Train,      psnr 24.44699055     loss = 0.04801134
Epoch 3     Valid,      psnr 23.70074201     loss = 0.05095885
Epoch 4     Train,      psnr 24.44699055     loss = 0.04801134
Epoch 4     Valid,      psnr 23.70074201     loss = 0.05095885
...
